plugins/CoreHome/templates/_menu.twig

```diff
@@ -1,4 +1,4 @@
-{% macro menu(menu, anchorlink, cssClass, currentModule, currentAction, collapsible) %}
+{% macro menu(menu, anchorlink, cssClass, currentModule, currentAction, collapsible, hasSomeAdminAccess) %}
```

Comment:
That doesn't look right. Why should that be shown to users with admin access, if the API method to fetch the count, actually checks for superuser permission?